src/azure-cli/azure/cli/command_modules/vm/tests/latest/recordings/test_vm_create_auto_create_subnet.yaml

```diff
@@ -57,7 +57,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.19.1 azsdk-python-azure-mgmt-network/17.1.0 Python/3.7.4 (Windows-10-10.0.19041-SP0)
```

Comment:
You upgrated network to 18.0.0. However, in yaml file
```
- AZURECLI/2.19.1 azsdk-python-azure-mgmt-network/17.1.0
```
Fake file. Request change.